airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_36_3_001__AddScheduleTypeToConfigsTable.java

```diff
@@ -0,0 +1,31 @@
+package io.airbyte.db.instance.configs.migrations;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class V0_36_3_001__AddScheduleTypeToConfigsTable extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_36_3_001__AddScheduleTypeToConfigsTable.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    // Warning: please do not use any jOOQ generated code to write a migration.
+    // As database schema changes, the generated jOOQ code can be deprecated. So
+    // old migration may not compile if there is any generated code.
+    final DSLContext ctx = DSL.using(context.getConnection());
+    addPublicColumn(ctx);
+  }
+
+  public static void addPublicColumn(final DSLContext ctx) {
+    ctx.alterTable("connection")
+        .addColumnIfNotExists(DSL.field("scheduleType", SQLDataType.VARCHAR(256).nullable(false)))
```

Comment:
You might need to have a default value for existing entries.

Reply:
+1, if this column is not nullable then we need to specify a default so that existing records have a value for this column.

I guess if there is an existing `schedule` field that is nullable though, it might be less obvious what an appropriate default for `scheduleType` would be.